pyb_utils/frame.py

```diff
@@ -54,25 +65,30 @@ def debug_frame(size, obj_uid, link_index):
     link_index : int
         Index of the link on the body to attach the frame to. Use `-1` for the
         base link.
+    Returns
+    -------
+    :
```

Comment:
nitpick: for return part of the docstring, the format is
```
Returns
-------
: <type>
    <description>
```
can you change that here and in the `debug_frame_world` docstring as well?

Reply:
fixed

Reply:
can you also remove the colon in front of the description part? there should only be a colon before the type

Reply:
sorry, removed now.

Reply:
thanks!